experimenter/experimenter/nimbus-ui/src/components/PageResults/index.tsx

```diff
@@ -463,10 +462,6 @@ const PageResults: React.FunctionComponent<RouteComponentProps> = () => {
                     className="rounded-bottom mb-3 border-top-0"
                     analysisBasis={selectedAnalysisBasis}
                     segment={selectedSegment}
-                    isDesktop={
-                      experiment.application ===
-                      NimbusExperimentApplicationEnum.DESKTOP
-                    }
```

Comment:
Nice that we can get rid of this.  So now we can abstract this way into just whether DAU exists or not, regardless of app, and that covers this case eh?

Reply:
Yea exactly. We had put in a special-case check for desktop because there was an improved metric specifically for desktop (QCDOU), but DAU supercedes that and is available for all apps.